test/parallel/test-http-server-connections-checking-leak.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// Flags: --expose-gc
+
+// Check that creating a server without listening is not leaking resources.
+
+require('../common');
+const onGC = require('../common/ongc');
+const Countdown = require('../common/countdown');
+
+const http = require('http');
+const max = 100;
+const countdown = new Countdown(max, () => {});
```

Comment:
There is a `common.mustCall()` for the callback in the `Countdown` implementation so it is ok, but it is not obvious to an external reader. Do you think it makes sense to make it explicit?

Reply:
I made it explicit by adding a comment.